apps/rush-lib/src/scripts/install-run.ts

```diff
@@ -409,7 +416,10 @@ export function installAndRun(
     {
       stdio: 'inherit',
       cwd: process.cwd(),
-      env: process.env
+      env: process.env,
+      // If binPath contains spaces, it must be quoted, and quoted paths will only be processed
+      // correctly if we use a shell (otherwise don't use a shell since it's a bit less efficient).
+      shell: binPath[0] === '"'
```

Comment:
I'm not sure there is an easy general solution here.  See [these notes](https://github.com/Microsoft/web-build-tools/blob/628ecb23e16638571db0ecc09b722714ddbe0b27/libraries/node-core-library/src/Executable.ts#L172) for some background.

But what happens if we simply set `shell: true` in all cases?  Or always when the OS is Windows?  Would that improve the situation you encountered without breaking other scenarios?